libglusterfs/src/glusterfs/glusterfs-fops.h

```diff
@@ -225,7 +225,8 @@ typedef enum gf_upcall_flags_t gf_upcall_flags_t;
 
 enum gf_dict_data_type_t {
     GF_DATA_TYPE_UNKNOWN = 0,
-    GF_DATA_TYPE_STR_OLD = 1,
+    /* Backward compatibility placeholder, do not use. */
+    GF_DATA_TYPE_STR_OLD = -1,
```

Comment:
I think the value shouldn't change, just to keep documented what was the old value. It won't be used anyway.